### ISSUE TITLE
Photo Detail Activity blocks fast scroll

### ISSUE THREAD

**Issue by felixWackernagel** [konifar/material-cat#31]
In the Detail Activity isn't it possible scroll fast from bottom to top. It stops below the image. Also a second fling gesture to reach the top edge of the image is blocking. Maybe its a bug in the design Support library?

**Comment by konifar** [comment]
@felixWackernagel Hi, I think so just like you said.
`CordinatorLayout` in Design Support Library does not scroll smoothly. It is problem of library.
I believe Design Support Library will be improved soon...

**Comment by felixWackernagel** [comment]
Hi Yusuke,
in Design Support Library v23 is the issue resolved. So hopefully in the
next time is everything ok.

https://code.google.com/p/android/issues/detail?id=177729&can=1&q=design%20support%20scroll&colspec=ID%20Type%20Status%20Owner%20Summary%20Stars

Am 19.08.2015 um 04:23 schrieb Yusuke Konishi:

**Comment by konifar** [comment]
@felixWackernagel Thanks for your information! So I will try to update to v23 and check it :)

**Comment by felixWackernagel** [comment]
Yes that would be cool. I looked at the revision diff and there was some
code changes. Can you update the app in the play store? I would like to
check it again.

Thanks
Felix

Am 19.08.2015 um 09:28 schrieb Yusuke Konishi:

**Comment by felixWackernagel** [comment]
Hello Again,

only for your interest. There is a new and more aggresive bug in v23.

The report contains a workaround. So check it out if you decide to
upgrade to version 23.
https://code.google.com/p/android/issues/detail?id=183166

I found the Design Support Library is totaly bugged and scares me to use
it in production.

Am 19.08.2015 um 09:28 schrieb Yusuke Konishi:

**Comment by konifar** [comment]
@felixWackernagel Hi, thanks for your information.
I'm so scared too... But I try to use Design Support Library because this project is my private one :)
